styles/styles.css

```diff
@@ -1101,6 +1102,16 @@ main .xxl-icons .icon {
   height: var(--icon-size-xxl);
 }
 
+main .xxxl-icons .icon {
+  width: var(--icon-size-xxxl);
+  height: var(--icon-size-xxxl);
+}
+
+main .block.columns.xxxl-icons .icon img {
+  width: var(--icon-size-xxxl);
+  height: var(--icon-size-xxxl);
+}
```

Comment:
That should be part of the columns stylesheet. Is it really needed though? Normally the `img` within the icon `span` should auto scale to it. Other than that LGTM